sys-libs/libhugetlbfs/libhugetlbfs-2.19-r1.ebuild

```diff
@@ -0,0 +1,158 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PERL_EXPORT_PHASE_FUNCTIONS="no"
+GENTOO_DEPEND_ON_PERL="no"
+PYTHON_COMPAT=( python2_7 )
+
+inherit eutils multilib perl-module python-any-r1 toolchain-funcs
+
+DESCRIPTION="easy hugepage access"
+HOMEPAGE="https://github.com/libhugetlbfs/libhugetlbfs"
+SRC_URI="https://github.com/libhugetlbfs/libhugetlbfs/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="static-libs test"
+
+DEPEND="test? ( ${PYTHON_DEPS} )"
+RDEPEND="dev-lang/perl:="
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-2.9-build.patch #332517
+	epatch "${FILESDIR}"/${PN}-2.6-noexec-stack.patch
+	epatch "${FILESDIR}"/${PN}-2.6-fixup-testsuite.patch
+
+	perl_set_version
+
+	sed -i \
+		-e '/^PREFIX/s:/local::' \
+		-e '1iBUILDTYPE = NATIVEONLY' \
+		-e '1iV = 1' \
+		-e "/^LIB\(32\)/s:=.*:= $(get_libdir):" \
+		-e '/^CC\(32\|64\)/s:=.*:= $(CC):' \
+		-e "/^PMDIR = .*\/perl5\/TLBC/s::PMDIR = ${VENDOR_LIB}\/TLBC:" \
+		Makefile || die "sed failed"
+	if [ "$(get_libdir)" == "lib64" ]; then
+		sed -i \
+			-e "/^LIB\(32\)/s:=.*:= lib32:" \
+				Makefile
+	fi
+}
+
+src_compile() {
+	tc-export AR
+	emake CC="$(tc-getCC)" libs tools
+}
+
+src_install() {
+	default
+	use static-libs || rm -f "${D}"/usr/$(get_libdir)/*.a
+	rm "${D}"/usr/bin/oprofile* || die
+}
+
+src_test_alloc_one() {
+	hugeadm="$1"
```

Comment:
Sorry, I don't understand what you mean with `local`. However I guess you are talking about the variables in `src_test_alloc_one` and `src_test`. I changed these variables to local variables.